ctconfig.yaml

```diff
@@ -0,0 +1,9 @@
+# See https://github.com/helm/chart-testing#configuration
+remote: origin
+target-branch: main
+chart-dirs:
+  - helm
+chart-repos:
+  - prometheus=https://prometheus-community.github.io/helm-charts
+  - grafana=https://grafana.github.io/helm-charts
+helm-extra-args: --timeout 600s
```

Comment:
Consider adding a newline at the end of the file. This is a common convention that helps with file processing in Unix-like systems. [medium]